samples/bluetooth/central_hr_coded/README.rst

```diff
@@ -26,6 +26,11 @@ The sample supports the following development kits:
    :header: heading
    :rows: nrf5340dk_nrf5340_cpuapp_and_cpuappns, nrf52840dongle_nrf52840, nrf52840dk_nrf52840
 
+.. note::
+
+  If you use nRF5340 DK, the relevant extra needed configuration of the network core will be taken from :file:`child_image` directory.
+  For more details see: :ref:`ug_multi_image_variables`.
```

Comment:
Proposing: Instead of the relevant extra needed -> the additional configuration.

Reply:
Done and also moved to the one file which is included in every place where it is needed.